src/transform/src/predicate_pushdown.rs

```diff
@@ -734,6 +735,31 @@ impl PredicatePushdown {
         }
     }
 
+    /// Remove predicates from Filter operators on top of Get operators
+    /// once the set of predicates that have been pushed down is known.
+    fn prune(
```

Comment:
Do this method and the block starting here do the same thing?
https://github.com/MaterializeInc/materialize/blob/311179db974fa36461f9574d6fe16bd1a6aa6e32/src/transform/src/predicate_pushdown.rs#L497

Reply:
hmmm, it seems so. I'll revert this and check why I added it.

Reply:
Reverted. You are completely right. Honesly idk why I added this :)